pkg/module_manager/global_hook_config.go

```diff
@@ -283,38 +291,37 @@ func NewHookConfigFromGoConfig(input *sdk.HookConfig) config.HookConfig {
 		monitor.WithFieldSelector(kubeCfg.FieldSelector)
 		monitor.WithNamespaceSelector(kubeCfg.NamespaceSelector)
 		monitor.WithLabelSelector(kubeCfg.LabelSelector)
-		monitor.JqFilter = kubeCfg.JqFilter
 		monitor.FilterFunc = kubeCfg.FilterFunc
 		// executeHookOnEvent is a priority
-		if kubeCfg.ExecuteHookOnEvents != nil {
-			monitor.WithEventTypes(kubeCfg.ExecuteHookOnEvents)
+		if kubeCfg.DontExecuteHookOnEvents {
+			monitor.WithEventTypes([]types2.WatchEventType{})
+		} else {
+			monitor.WithEventTypes(nil)
 		}
 
 		kubeConfig := OnKubernetesEventConfig{}
 		kubeConfig.Monitor = monitor
-		kubeConfig.AllowFailure = kubeCfg.AllowFailure
+		kubeConfig.AllowFailure = input.AllowFailure
 		if kubeCfg.Name == "" {
-			kubeConfig.BindingName = string(OnKubernetesEvent)
-		} else {
-			kubeConfig.BindingName = kubeCfg.Name
+			return c, spew.Errorf(`"name" is a required field in binding: %v`)
 		}
-		kubeConfig.IncludeSnapshotsFrom = kubeCfg.IncludeSnapshotsFrom
-		if kubeCfg.Queue == "" {
+		kubeConfig.BindingName = kubeCfg.Name
+		if input.Queue == "" {
 			kubeConfig.Queue = "main"
 		} else {
-			kubeConfig.Queue = kubeCfg.Queue
+			kubeConfig.Queue = input.Queue
 		}
-		kubeConfig.Group = kubeCfg.Group
+		kubeConfig.Group = "main"
```

Comment:
"Have to"? I disagree. We'll remove such logic from existing hooks?

Reply:
It may be tricky to get rid of such logic. But if you say so, ok, I'm fine with it. The only thing I would like to suggest to move the default group name to a constant variable.

Reply:
>It may be tricky

We'll apply our best brute-force technique

>move the default group name to a constant variable

Done.